packages/keychain/src/components/connect/Login.tsx

```diff
@@ -81,17 +81,10 @@ function Form({
         credentialId,
       });
 
-      switch (mode) {
-        case LoginMode.Webauthn:
-          await doLogin(usernameField.value, credentialId);
-          break;
-        case LoginMode.Controller:
-          if (policies.length === 0) {
-            throw new Error("Policies required for controller ");
-          }
-
-          await controller.approve(origin, expiresAt, policies);
-          break;
+      if (mode === LoginMode.Controller && policies.length > 0) {
+        await controller.approve(origin, expiresAt, policies);
+      } else {
+        await doLogin(usernameField.value, credentialId);
```

Comment:
@JunichiSugiura I made small change here to make login require webauthn authentication if no policies. Otherwise ppl can log into others account, any txn will fail but just feels weird.